class/LaserLlama; Alternate Rogue.json

```diff
@@ -6176,7 +6177,7 @@
 					"type": "table",
 					"caption": "Bewildering Blow",
 					"colLabels": [
-						"1d10",
```

Comment:
fwiw both the dice header _and_ the numbers in the first column should get auto-detected; you shouldn't need to specify `@dice` in the header + `cell`s 🤔 Did it not work without these?

Reply:
Oh, it probably did  i don't think I checked. I was more following the format of the Render Demo (I reference it occasionally when writing up homebrew stuff)